Auto-detected KUBERNETES_SERVICE_DNS_DOMAIN: cluster.local
+ exec /usr/bin/tini -w -e 143 -- java -Dlog4j2.configurationFile=file:/opt/strimzi/custom-config/log4j2.properties -Dvertx.cacheDirBase=/tmp/vertx-cache -Djava.security.egd=file:/dev/./urandom -XX:+ExitOnOutOfMemoryError -XX:MinRAMPercentage=15 -XX:MaxRAMPercentage=20 -classpath :lib/io.strimzi.cluster-operator-0.43.0.jar:lib/io.strimzi.api-0.43.0.jar:lib/io.fabric8.kubernetes-model-apiextensions-6.13.3.jar:lib/io.strimzi.config-model-0.43.0.jar:lib/io.strimzi.operator-common-0.43.0.jar:lib/io.strimzi.crd-annotations-0.43.0.jar:lib/org.quartz-scheduler.quartz-2.3.2.jar:lib/org.eclipse.jetty.jetty-server-9.4.54.v20240208.jar:lib/org.eclipse.jetty.jetty-http-9.4.54.v20240208.jar:lib/org.eclipse.jetty.jetty-io-9.4.54.v20240208.jar:lib/javax.servlet.javax.servlet-api-3.1.0.jar:lib/io.fabric8.openshift-client-6.13.3.jar:lib/com.github.mifmif.generex-1.0.2.jar:lib/dk.brics.automaton.automaton-1.11-8.jar:lib/io.fabric8.openshift-client-api-6.13.3.jar:lib/io.fabric8.openshift-model-clusterautoscaling-6.13.3.jar:lib/io.fabric8.openshift-model-operator-6.13.3.jar:lib/io.fabric8.openshift-model-operatorhub-6.13.3.jar:lib/io.fabric8.openshift-model-machine-6.13.3.jar:lib/io.fabric8.openshift-model-whereabouts-6.13.3.jar:lib/io.fabric8.openshift-model-monitoring-6.13.3.jar:lib/io.fabric8.openshift-model-storageversionmigrator-6.13.3.jar:lib/io.fabric8.openshift-model-tuned-6.13.3.jar:lib/io.fabric8.openshift-model-console-6.13.3.jar:lib/io.fabric8.openshift-model-config-6.13.3.jar:lib/io.fabric8.openshift-model-machineconfig-6.13.3.jar:lib/io.fabric8.openshift-model-miscellaneous-6.13.3.jar:lib/io.fabric8.openshift-model-hive-6.13.3.jar:lib/io.fabric8.openshift-model-installer-6.13.3.jar:lib/io.fabric8.kubernetes-client-6.13.3.jar:lib/io.fabric8.kubernetes-httpclient-jdk-6.13.3.jar:lib/io.fabric8.kubernetes-client-api-6.13.3.jar:lib/io.fabric8.kubernetes-model-gatewayapi-6.13.3.jar:lib/io.fabric8.kubernetes-model-resource-6.13.3.jar:lib/io.fabric8.kubernetes-model-admissionregistration-6.13.3.jar:lib/io.fabric8.kubernetes-model-autoscaling-6.13.3.jar:lib/io.fabric8.kubernetes-model-batch-6.13.3.jar:lib/io.fabric8.kubernetes-model-certificates-6.13.3.jar:lib/io.fabric8.kubernetes-model-discovery-6.13.3.jar:lib/io.fabric8.kubernetes-model-extensions-6.13.3.jar:lib/io.fabric8.kubernetes-model-flowcontrol-6.13.3.jar:lib/io.fabric8.kubernetes-model-metrics-6.13.3.jar:lib/io.fabric8.kubernetes-model-scheduling-6.13.3.jar:lib/io.fabric8.kubernetes-model-node-6.13.3.jar:lib/org.slf4j.slf4j-api-1.7.36.jar:lib/org.snakeyaml.snakeyaml-engine-2.7.jar:lib/com.fasterxml.jackson.datatype.jackson-datatype-jsr310-2.16.2.jar:lib/io.fabric8.kubernetes-model-core-6.13.3.jar:lib/io.fabric8.kubernetes-model-common-6.13.3.jar:lib/io.fabric8.kubernetes-model-events-6.13.3.jar:lib/io.fabric8.kubernetes-model-policy-6.13.3.jar:lib/io.fabric8.kubernetes-model-rbac-6.13.3.jar:lib/io.fabric8.kubernetes-model-apps-6.13.3.jar:lib/io.fabric8.kubernetes-model-storageclass-6.13.3.jar:lib/io.fabric8.kubernetes-model-networking-6.13.3.jar:lib/io.fabric8.kubernetes-model-coordination-6.13.3.jar:lib/io.fabric8.openshift-model-6.13.3.jar:lib/io.fabric8.zjsonpatch-0.3.0.jar:lib/com.fasterxml.jackson.core.jackson-core-2.16.2.jar:lib/com.fasterxml.jackson.core.jackson-databind-2.16.2.jar:lib/com.fasterxml.jackson.core.jackson-annotations-2.16.2.jar:lib/com.fasterxml.jackson.dataformat.jackson-dataformat-yaml-2.16.2.jar:lib/org.yaml.snakeyaml-2.2.jar:lib/io.vertx.vertx-core-4.5.9.jar:lib/io.netty.netty-common-4.1.111.Final.jar:lib/io.netty.netty-buffer-4.1.111.Final.jar:lib/io.netty.netty-handler-4.1.111.Final.jar:lib/io.netty.netty-handler-proxy-4.1.111.Final.jar:lib/io.netty.netty-codec-http-4.1.111.Final.jar:lib/io.netty.netty-codec-http2-4.1.111.Final.jar:lib/io.netty.netty-resolver-4.1.111.Final.jar:lib/io.netty.netty-resolver-dns-4.1.111.Final.jar:lib/io.netty.netty-codec-dns-4.1.111.Final.jar:lib/io.strimzi.certificate-manager-0.43.0.jar:lib/org.apache.logging.log4j.log4j-core-2.17.2.jar:lib/org.apache.logging.log4j.log4j-api-2.17.2.jar:lib/org.apache.logging.log4j.log4j-slf4j-impl-2.17.2.jar:lib/io.strimzi.kafka-oauth-server-0.15.0.jar:lib/io.strimzi.kafka-oauth-server-plain-0.15.0.jar:lib/io.strimzi.kafka-oauth-client-0.15.0.jar:lib/io.strimzi.kafka-oauth-common-0.15.0.jar:lib/com.nimbusds.nimbus-jose-jwt-9.37.2.jar:lib/com.github.stephenc.jcip.jcip-annotations-1.0-1.jar:lib/com.jayway.jsonpath.json-path-2.9.0.jar:lib/net.minidev.json-smart-2.5.0.jar:lib/net.minidev.accessors-smart-2.5.0.jar:lib/org.eclipse.jetty.jetty-util-9.4.54.v20240208.jar:lib/org.apache.kafka.kafka-clients-3.8.0.jar:lib/com.github.luben.zstd-jni-1.5.6-3.jar:lib/org.lz4.lz4-java-1.8.0.jar:lib/org.apache.kafka.kafka-server-common-3.8.0.jar:lib/com.yammer.metrics.metrics-core-2.2.0.jar:lib/net.sf.jopt-simple.jopt-simple-5.0.4.jar:lib/org.pcollections.pcollections-4.0.1.jar:lib/org.apache.zookeeper.zookeeper-3.8.4.jar:lib/org.apache.yetus.audience-annotations-0.12.0.jar:lib/commons-io.commons-io-2.11.0.jar:lib/org.apache.zookeeper.zookeeper-jute-3.8.4.jar:lib/io.netty.netty-transport-4.1.111.Final.jar:lib/io.netty.netty-transport-native-epoll-4.1.111.Final-linux-x86_64.jar:lib/io.netty.netty-transport-native-unix-common-4.1.111.Final.jar:lib/io.netty.netty-transport-classes-epoll-4.1.111.Final.jar:lib/io.netty.netty-transport-native-epoll-4.1.111.Final-linux-aarch_64.jar:lib/io.micrometer.micrometer-core-1.12.3.jar:lib/io.micrometer.micrometer-commons-1.12.3.jar:lib/io.micrometer.micrometer-observation-1.12.3.jar:lib/org.hdrhistogram.HdrHistogram-2.1.12.jar:lib/org.latencyutils.LatencyUtils-2.0.3.jar:lib/io.micrometer.micrometer-registry-prometheus-1.12.3.jar:lib/io.prometheus.simpleclient_common-0.16.0.jar:lib/io.prometheus.simpleclient-0.16.0.jar:lib/io.prometheus.simpleclient_tracer_otel-0.16.0.jar:lib/io.prometheus.simpleclient_tracer_common-0.16.0.jar:lib/io.prometheus.simpleclient_tracer_otel_agent-0.16.0.jar:lib/io.vertx.vertx-micrometer-metrics-4.5.9.jar:lib/io.netty.netty-codec-4.1.111.Final.jar:lib/io.netty.netty-codec-socks-4.1.111.Final.jar io.strimzi.operator.cluster.Main
2025-09-22 16:32:29 INFO  Main:60 - ClusterOperator 0.43.0 is starting
2025-09-22 16:32:29 INFO  Util:196 - Using config:
	PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
	container: oci
	STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE: quay.io/strimzi/operator:0.43.0
	JAVA_OPTS:  -Dlog4j2.configurationFile=file:/opt/strimzi/custom-config/log4j2.properties -Dvertx.cacheDirBase=/tmp/vertx-cache -Djava.security.egd=file:/dev/./urandom  -XX:+ExitOnOutOfMemoryError -XX:MinRAMPercentage=15 -XX:MaxRAMPercentage=20
	NSS_SDB_USE_CACHE: no
	STRIMZI_HOME: /opt/strimzi
	PWD: /opt/strimzi
	STRIMZI_LEADER_ELECTION_IDENTITY: strimzi-cluster-operator-6f6d7f4bc7-6gr6m
	JAVA_MAIN: io.strimzi.operator.cluster.Main
	KUBERNETES_PORT_443_TCP: tcp://172.30.0.1:443
	STRIMZI_LEADER_ELECTION_ENABLED: true
	STRIMZI_VERSION: 0.43.0
	STRIMZI_DEFAULT_KANIKO_EXECUTOR_IMAGE: quay.io/strimzi/kaniko-executor:0.43.0
	STRIMZI_NAMESPACE: ksql043
	TINI_SHA256_S390X: 931b70a182af879ca249ae9de87ef68423121b38d235c78997fafc680ceab32d
	STRIMZI_DEFAULT_MAVEN_BUILDER: quay.io/strimzi/maven-builder:0.43.0
	STRIMZI_FEATURE_GATES: 
	TINI_SHA256_AMD64: 93dcc18adc78c65a028a84799ecf8ad40c936fdfc5f2a57b1acda5a8117fa82c
	KUBERNETES_SERVICE_DNS_DOMAIN: cluster.local
	STRIMZI_LEADER_ELECTION_LEASE_NAME: strimzi-cluster-operator
	KUBERNETES_SERVICE_PORT_HTTPS: 443
	SHLVL: 0
	STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE: quay.io/strimzi/kafka-bridge:0.30.0
	STRIMZI_KAFKA_MIRROR_MAKER_IMAGES: 3.7.0=quay.io/strimzi/kafka:0.43.0-kafka-3.7.0
3.7.1=quay.io/strimzi/kafka:0.43.0-kafka-3.7.1
3.8.0=quay.io/strimzi/kafka:0.43.0-kafka-3.8.0

	KUBERNETES_PORT: tcp://172.30.0.1:443
	JAVA_HOME: /usr/lib/jvm/jre-17
	TERM: xterm
	STRIMZI_FULL_RECONCILIATION_INTERVAL_MS: 120000
	KUBERNETES_SERVICE_HOST: 172.30.0.1
	JAVA_CLASSPATH: :lib/io.strimzi.cluster-operator-0.43.0.jar:lib/io.strimzi.api-0.43.0.jar:lib/io.fabric8.kubernetes-model-apiextensions-6.13.3.jar:lib/io.strimzi.config-model-0.43.0.jar:lib/io.strimzi.operator-common-0.43.0.jar:lib/io.strimzi.crd-annotations-0.43.0.jar:lib/org.quartz-scheduler.quartz-2.3.2.jar:lib/org.eclipse.jetty.jetty-server-9.4.54.v20240208.jar:lib/org.eclipse.jetty.jetty-http-9.4.54.v20240208.jar:lib/org.eclipse.jetty.jetty-io-9.4.54.v20240208.jar:lib/javax.servlet.javax.servlet-api-3.1.0.jar:lib/io.fabric8.openshift-client-6.13.3.jar:lib/com.github.mifmif.generex-1.0.2.jar:lib/dk.brics.automaton.automaton-1.11-8.jar:lib/io.fabric8.openshift-client-api-6.13.3.jar:lib/io.fabric8.openshift-model-clusterautoscaling-6.13.3.jar:lib/io.fabric8.openshift-model-operator-6.13.3.jar:lib/io.fabric8.openshift-model-operatorhub-6.13.3.jar:lib/io.fabric8.openshift-model-machine-6.13.3.jar:lib/io.fabric8.openshift-model-whereabouts-6.13.3.jar:lib/io.fabric8.openshift-model-monitoring-6.13.3.jar:lib/io.fabric8.openshift-model-storageversionmigrator-6.13.3.jar:lib/io.fabric8.openshift-model-tuned-6.13.3.jar:lib/io.fabric8.openshift-model-console-6.13.3.jar:lib/io.fabric8.openshift-model-config-6.13.3.jar:lib/io.fabric8.openshift-model-machineconfig-6.13.3.jar:lib/io.fabric8.openshift-model-miscellaneous-6.13.3.jar:lib/io.fabric8.openshift-model-hive-6.13.3.jar:lib/io.fabric8.openshift-model-installer-6.13.3.jar:lib/io.fabric8.kubernetes-client-6.13.3.jar:lib/io.fabric8.kubernetes-httpclient-jdk-6.13.3.jar:lib/io.fabric8.kubernetes-client-api-6.13.3.jar:lib/io.fabric8.kubernetes-model-gatewayapi-6.13.3.jar:lib/io.fabric8.kubernetes-model-resource-6.13.3.jar:lib/io.fabric8.kubernetes-model-admissionregistration-6.13.3.jar:lib/io.fabric8.kubernetes-model-autoscaling-6.13.3.jar:lib/io.fabric8.kubernetes-model-batch-6.13.3.jar:lib/io.fabric8.kubernetes-model-certificates-6.13.3.jar:lib/io.fabric8.kubernetes-model-discovery-6.13.3.jar:lib/io.fabric8.kubernetes-model-extensions-6.13.3.jar:lib/io.fabric8.kubernetes-model-flowcontrol-6.13.3.jar:lib/io.fabric8.kubernetes-model-metrics-6.13.3.jar:lib/io.fabric8.kubernetes-model-scheduling-6.13.3.jar:lib/io.fabric8.kubernetes-model-node-6.13.3.jar:lib/org.slf4j.slf4j-api-1.7.36.jar:lib/org.snakeyaml.snakeyaml-engine-2.7.jar:lib/com.fasterxml.jackson.datatype.jackson-datatype-jsr310-2.16.2.jar:lib/io.fabric8.kubernetes-model-core-6.13.3.jar:lib/io.fabric8.kubernetes-model-common-6.13.3.jar:lib/io.fabric8.kubernetes-model-events-6.13.3.jar:lib/io.fabric8.kubernetes-model-policy-6.13.3.jar:lib/io.fabric8.kubernetes-model-rbac-6.13.3.jar:lib/io.fabric8.kubernetes-model-apps-6.13.3.jar:lib/io.fabric8.kubernetes-model-storageclass-6.13.3.jar:lib/io.fabric8.kubernetes-model-networking-6.13.3.jar:lib/io.fabric8.kubernetes-model-coordination-6.13.3.jar:lib/io.fabric8.openshift-model-6.13.3.jar:lib/io.fabric8.zjsonpatch-0.3.0.jar:lib/com.fasterxml.jackson.core.jackson-core-2.16.2.jar:lib/com.fasterxml.jackson.core.jackson-databind-2.16.2.jar:lib/com.fasterxml.jackson.core.jackson-annotations-2.16.2.jar:lib/com.fasterxml.jackson.dataformat.jackson-dataformat-yaml-2.16.2.jar:lib/org.yaml.snakeyaml-2.2.jar:lib/io.vertx.vertx-core-4.5.9.jar:lib/io.netty.netty-common-4.1.111.Final.jar:lib/io.netty.netty-buffer-4.1.111.Final.jar:lib/io.netty.netty-handler-4.1.111.Final.jar:lib/io.netty.netty-handler-proxy-4.1.111.Final.jar:lib/io.netty.netty-codec-http-4.1.111.Final.jar:lib/io.netty.netty-codec-http2-4.1.111.Final.jar:lib/io.netty.netty-resolver-4.1.111.Final.jar:lib/io.netty.netty-resolver-dns-4.1.111.Final.jar:lib/io.netty.netty-codec-dns-4.1.111.Final.jar:lib/io.strimzi.certificate-manager-0.43.0.jar:lib/org.apache.logging.log4j.log4j-core-2.17.2.jar:lib/org.apache.logging.log4j.log4j-api-2.17.2.jar:lib/org.apache.logging.log4j.log4j-slf4j-impl-2.17.2.jar:lib/io.strimzi.kafka-oauth-server-0.15.0.jar:lib/io.strimzi.kafka-oauth-server-plain-0.15.0.jar:lib/io.strimzi.kafka-oauth-client-0.15.0.jar:lib/io.strimzi.kafka-oauth-common-0.15.0.jar:lib/com.nimbusds.nimbus-jose-jwt-9.37.2.jar:lib/com.github.stephenc.jcip.jcip-annotations-1.0-1.jar:lib/com.jayway.jsonpath.json-path-2.9.0.jar:lib/net.minidev.json-smart-2.5.0.jar:lib/net.minidev.accessors-smart-2.5.0.jar:lib/org.eclipse.jetty.jetty-util-9.4.54.v20240208.jar:lib/org.apache.kafka.kafka-clients-3.8.0.jar:lib/com.github.luben.zstd-jni-1.5.6-3.jar:lib/org.lz4.lz4-java-1.8.0.jar:lib/org.apache.kafka.kafka-server-common-3.8.0.jar:lib/com.yammer.metrics.metrics-core-2.2.0.jar:lib/net.sf.jopt-simple.jopt-simple-5.0.4.jar:lib/org.pcollections.pcollections-4.0.1.jar:lib/org.apache.zookeeper.zookeeper-3.8.4.jar:lib/org.apache.yetus.audience-annotations-0.12.0.jar:lib/commons-io.commons-io-2.11.0.jar:lib/org.apache.zookeeper.zookeeper-jute-3.8.4.jar:lib/io.netty.netty-transport-4.1.111.Final.jar:lib/io.netty.netty-transport-native-epoll-4.1.111.Final-linux-x86_64.jar:lib/io.netty.netty-transport-native-unix-common-4.1.111.Final.jar:lib/io.netty.netty-transport-classes-epoll-4.1.111.Final.jar:lib/io.netty.netty-transport-native-epoll-4.1.111.Final-linux-aarch_64.jar:lib/io.micrometer.micrometer-core-1.12.3.jar:lib/io.micrometer.micrometer-commons-1.12.3.jar:lib/io.micrometer.micrometer-observation-1.12.3.jar:lib/org.hdrhistogram.HdrHistogram-2.1.12.jar:lib/org.latencyutils.LatencyUtils-2.0.3.jar:lib/io.micrometer.micrometer-registry-prometheus-1.12.3.jar:lib/io.prometheus.simpleclient_common-0.16.0.jar:lib/io.prometheus.simpleclient-0.16.0.jar:lib/io.prometheus.simpleclient_tracer_otel-0.16.0.jar:lib/io.prometheus.simpleclient_tracer_common-0.16.0.jar:lib/io.prometheus.simpleclient_tracer_otel_agent-0.16.0.jar:lib/io.vertx.vertx-micrometer-metrics-4.5.9.jar:lib/io.netty.netty-codec-4.1.111.Final.jar:lib/io.netty.netty-codec-socks-4.1.111.Final.jar
	STRIMZI_DEFAULT_CRUISE_CONTROL_IMAGE: quay.io/strimzi/kafka:0.43.0-kafka-3.8.0
	TINI_VERSION: v0.19.0
	STRIMZI_OPERATION_TIMEOUT_MS: 300000
	KUBERNETES_PORT_443_TCP_ADDR: 172.30.0.1
	STRIMZI_LEADER_ELECTION_LEASE_NAMESPACE: ksql043
	STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES: 3.7.0=quay.io/strimzi/kafka:0.43.0-kafka-3.7.0
3.7.1=quay.io/strimzi/kafka:0.43.0-kafka-3.7.1
3.8.0=quay.io/strimzi/kafka:0.43.0-kafka-3.8.0

	KUBERNETES_PORT_443_TCP_PROTO: tcp
	STRIMZI_DEFAULT_USER_OPERATOR_IMAGE: quay.io/strimzi/operator:0.43.0
	KUBERNETES_SERVICE_PORT: 443
	STRIMZI_DEFAULT_KAFKA_EXPORTER_IMAGE: quay.io/strimzi/kafka:0.43.0-kafka-3.8.0
	TINI_SHA256_ARM64: 07952557df20bfd2a95f9bef198b445e006171969499a1d361bd9e6f8e5e0e81
	STRIMZI_DEFAULT_KAFKA_INIT_IMAGE: quay.io/strimzi/operator:0.43.0
	TINI_SHA256_PPC64LE: 3f658420974768e40810001a038c29d003728c5fe86da211cff5059e48cfdfde
	HOSTNAME: strimzi-cluster-operator-6f6d7f4bc7-6gr6m
	STRIMZI_KAFKA_CONNECT_IMAGES: 3.7.0=quay.io/strimzi/kafka:0.43.0-kafka-3.7.0
3.7.1=quay.io/strimzi/kafka:0.43.0-kafka-3.7.1
3.8.0=quay.io/strimzi/kafka:0.43.0-kafka-3.8.0

	STRIMZI_KAFKA_IMAGES: 3.7.0=quay.io/strimzi/kafka:0.43.0-kafka-3.7.0
3.7.1=quay.io/strimzi/kafka:0.43.0-kafka-3.7.1
3.8.0=quay.io/strimzi/kafka:0.43.0-kafka-3.8.0

	KUBERNETES_PORT_443_TCP_PORT: 443
	STRIMZI_OPERATOR_NAME: strimzi-cluster-operator-6f6d7f4bc7-6gr6m
	HOME: /
	STRIMZI_OPERATOR_NAMESPACE: ksql043
	MALLOC_ARENA_MAX: 2

2025-09-22 16:32:30 INFO  Main:63 - Cluster Operator configuration is ClusterOperatorConfig{
	namespaces='[ksql043]'
	reconciliationIntervalMs=120000
	operationTimeoutMs=300000
	connectBuildTimeoutMs=300000
	networkPolicyGeneration=true
	versions='versions{3.7.0={proto: 3.7 msg: 3.7 kafka-image: quay.io/strimzi/kafka:0.43.0-kafka-3.7.0 connect-image: quay.io/strimzi/kafka:0.43.0-kafka-3.7.0 mirrormaker-image: quay.io/strimzi/kafka:0.43.0-kafka-3.7.0 mirrormaker2-image: quay.io/strimzi/kafka:0.43.0-kafka-3.7.0}, 3.7.1={proto: 3.7 msg: 3.7 kafka-image: quay.io/strimzi/kafka:0.43.0-kafka-3.7.1 connect-image: quay.io/strimzi/kafka:0.43.0-kafka-3.7.1 mirrormaker-image: quay.io/strimzi/kafka:0.43.0-kafka-3.7.1 mirrormaker2-image: quay.io/strimzi/kafka:0.43.0-kafka-3.7.1}, 3.8.0={proto: 3.8 msg: 3.8 kafka-image: quay.io/strimzi/kafka:0.43.0-kafka-3.8.0 connect-image: quay.io/strimzi/kafka:0.43.0-kafka-3.8.0 mirrormaker-image: quay.io/strimzi/kafka:0.43.0-kafka-3.8.0 mirrormaker2-image: quay.io/strimzi/kafka:0.43.0-kafka-3.8.0}}'
	imagePullPolicy='null'
	imagePullSecrets='null'
	operatorNamespace='ksql043'
	operatorNamespaceLabels='null'
	customResourceSelector='null'
	featureGates='FeatureGates(ContinueReconciliationOnManualRollingUpdateFailure=false)'
	zkAdminSessionTimeoutMs=10000
	dnsCacheTtlSec=30
	podSetReconciliationOnly=false
	podSetControllerWorkQueueSize=1024
	operatorName='strimzi-cluster-operator-6f6d7f4bc7-6gr6m'
	podSecurityProviderClass='io.strimzi.plugin.security.profiles.impl.BaselinePodSecurityProvider'
	leaderElectionConfig='LeaderElectionConfig{leaseName='strimzi-cluster-operator', namespace='ksql043', identity='strimzi-cluster-operator-6f6d7f4bc7-6gr6m', leaseDuration=PT15S, renewDeadline=PT10S, retryPeriod=PT2S}'}
2025-09-22 16:32:33 INFO  Main:278 - Health and metrics server is ready on port 8080)
2025-09-22 16:32:33 INFO  Main:242 - Waiting to become a leader
2025-09-22 16:32:33 INFO  LeaderElectionManager:69 - Starting the Leader Elector
2025-09-22 16:32:34 INFO  LeaderElectionManager:121 - The new leader is strimzi-cluster-operator-6f6d7f4bc7-6gr6m
2025-09-22 16:32:34 INFO  LeaderElectionManager:103 - Started being a leader
2025-09-22 16:32:34 INFO  Main:224 - I'm the new leader
2025-09-22 16:32:34 WARN  PlatformFeaturesAvailability:156 - API Group route.openshift.io is supported
2025-09-22 16:32:34 WARN  PlatformFeaturesAvailability:156 - API Group build.openshift.io is supported
2025-09-22 16:32:34 WARN  PlatformFeaturesAvailability:156 - API Group image.openshift.io is supported
2025-09-22 16:32:34 INFO  Main:111 - Environment facts gathered: PlatformFeaturesAvailability(KubernetesVersion=1.32,OpenShiftRoutes=true,OpenShiftBuilds=true,OpenShiftImageStreams=true)
2025-09-22 16:32:35 INFO  PodSecurityProviderFactory:43 - Found PodSecurityProvider io.strimzi.plugin.security.profiles.impl.BaselinePodSecurityProvider
2025-09-22 16:32:35 INFO  PodSecurityProviderFactory:62 - Initializing PodSecurityProvider io.strimzi.plugin.security.profiles.impl.BaselinePodSecurityProvider
2025-09-22 16:32:35 INFO  ClusterOperator:86 - Creating ClusterOperator for namespace ksql043
2025-09-22 16:32:35 INFO  ClusterOperator:100 - Starting ClusterOperator for namespace ksql043
2025-09-22 16:32:35 INFO  StrimziPodSetController:583 - Starting the StrimziPodSet controller
2025-09-22 16:32:35 INFO  StrimziPodSetController:552 - Starting StrimziPodSet controller for namespace ksql043
2025-09-22 16:32:35 WARN  VersionUsageUtils:60 - The client is using resource type 'strimzipodsets' with unstable version 'v1beta2'
2025-09-22 16:32:35 WARN  VersionUsageUtils:60 - The client is using resource type 'kafkas' with unstable version 'v1beta2'
2025-09-22 16:32:35 WARN  VersionUsageUtils:60 - The client is using resource type 'kafkaconnects' with unstable version 'v1beta2'
2025-09-22 16:32:35 WARN  VersionUsageUtils:60 - The client is using resource type 'kafkamirrormaker2s' with unstable version 'v1beta2'
2025-09-22 16:32:35 INFO  StrimziPodSetController:555 - Waiting for informers to sync
2025-09-22 16:32:39 INFO  ClusterOperator:114 - Opened watch for Kafka operator
2025-09-22 16:32:39 INFO  StrimziPodSetController:560 - Informers are in-sync
2025-09-22 16:32:40 INFO  ClusterOperator:114 - Opened watch for KafkaMirrorMaker operator
2025-09-22 16:32:40 INFO  ClusterOperator:114 - Opened watch for KafkaConnect operator
2025-09-22 16:32:40 INFO  ClusterOperator:114 - Opened watch for KafkaBridge operator
2025-09-22 16:32:40 INFO  ClusterOperator:114 - Opened watch for KafkaMirrorMaker2 operator
2025-09-22 16:32:40 INFO  ClusterOperator:114 - Opened watch for KafkaRebalance operator
2025-09-22 16:32:40 INFO  ClusterOperator:122 - Opened watch for KafkaNodePool operator
2025-09-22 16:32:40 INFO  ClusterOperator:129 - Opened watch for KafkaConnector operator
2025-09-22 16:32:40 INFO  ClusterOperator:137 - Setting up periodic reconciliation for namespace ksql043
2025-09-22 16:32:40 INFO  Main:193 - Cluster Operator verticle started in namespace ksql043 without label selector
2025-09-22 16:33:03 INFO  AbstractOperator:510 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): Kafka my-cluster in namespace ksql043 was ADDED
2025-09-22 16:33:03 INFO  KafkaAssemblyOperator:920 - Reconciliation #2(watch) Kafka(ksql043/my-cluster): KafkaNodePool dual-role in namespace ksql043 was ADDED
2025-09-22 16:33:03 INFO  AbstractOperator:264 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:33:03 INFO  AbstractOperator:510 - Reconciliation #3(watch) Kafka(ksql043/my-cluster): Kafka my-cluster in namespace ksql043 was MODIFIED
2025-09-22 16:33:03 INFO  CrdOperator:123 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): Status of Kafka my-cluster in namespace ksql043 has been updated
2025-09-22 16:33:03 INFO  Ca:1026 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): Generating CA with subject=Subject(organizationName='io.strimzi', commonName='cluster-ca v0', dnsNames=[], ipAddresses=[])
2025-09-22 16:33:05 INFO  Ca:1026 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): Generating CA with subject=Subject(organizationName='io.strimzi', commonName='clients-ca v0', dnsNames=[], ipAddresses=[])
2025-09-22 16:33:08 INFO  Ca:412 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): Generating certificate Subject(organizationName='io.strimzi', commonName='cluster-operator', dnsNames=[], ipAddresses=[]), signed by CA cluster-ca
2025-09-22 16:33:09 WARN  VersionUsageUtils:60 - The client is using resource type 'kafkanodepools' with unstable version 'v1beta2'
2025-09-22 16:33:09 ERROR AbstractOperator:283 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:33:09 INFO  AbstractOperator:510 - Reconciliation #4(watch) Kafka(ksql043/my-cluster): Kafka my-cluster in namespace ksql043 was MODIFIED
2025-09-22 16:33:09 INFO  CrdOperator:123 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): Status of Kafka my-cluster in namespace ksql043 has been updated
2025-09-22 16:33:09 INFO  AbstractOperator:264 - Reconciliation #2(watch) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:33:09 WARN  AbstractOperator:556 - Reconciliation #1(watch) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:33:10 ERROR AbstractOperator:283 - Reconciliation #2(watch) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:33:10 WARN  AbstractOperator:556 - Reconciliation #2(watch) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:33:10 INFO  AbstractOperator:264 - Reconciliation #3(watch) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:33:11 ERROR AbstractOperator:283 - Reconciliation #3(watch) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:33:11 WARN  AbstractOperator:556 - Reconciliation #3(watch) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:33:11 INFO  AbstractOperator:264 - Reconciliation #4(watch) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:33:11 ERROR AbstractOperator:283 - Reconciliation #4(watch) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:33:11 WARN  AbstractOperator:556 - Reconciliation #4(watch) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:34:00 WARN  LeaderElector:216 - Exception occurred while acquiring lock 'LeaseLock: ksql043 - strimzi-cluster-operator (strimzi-cluster-operator-6f6d7f4bc7-6gr6m) retrying...'
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: PATCH at: https://172.30.0.1:443/apis/coordination.k8s.io/v1/namespaces/ksql043/leases/strimzi-cluster-operator. Message: Operation cannot be fulfilled on leases.coordination.k8s.io "strimzi-cluster-operator": the object has been modified; please apply your changes to the latest version and try again. Received status: Status(apiVersion=v1, code=409, details=StatusDetails(causes=[], group=coordination.k8s.io, kind=leases, name=strimzi-cluster-operator, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=Operation cannot be fulfilled on leases.coordination.k8s.io "strimzi-cluster-operator": the object has been modified; please apply your changes to the latest version and try again, metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Conflict, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handlePatch(OperationSupport.java:419) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handlePatch(OperationSupport.java:397) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handlePatch(BaseOperation.java:764) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.HasMetadataOperation.lambda$patch$2(HasMetadataOperation.java:231) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.HasMetadataOperation.patch(HasMetadataOperation.java:236) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.HasMetadataOperation.patch(HasMetadataOperation.java:251) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.HasMetadataOperation.patch(HasMetadataOperation.java:44) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.extension.ResourceAdapter.patch(ResourceAdapter.java:305) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.extended.leaderelection.resourcelock.ResourceLock.update(ResourceLock.java:64) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.extended.leaderelection.LeaderElector.tryAcquireOrRenew(LeaderElector.java:247) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.extended.leaderelection.LeaderElector.lambda$renewWithTimeout$6(LeaderElector.java:207) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.extended.leaderelection.LeaderElector.lambda$loop$8(LeaderElector.java:292) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: PATCH at: https://172.30.0.1:443/apis/coordination.k8s.io/v1/namespaces/ksql043/leases/strimzi-cluster-operator. Message: Operation cannot be fulfilled on leases.coordination.k8s.io "strimzi-cluster-operator": the object has been modified; please apply your changes to the latest version and try again. Received status: Status(apiVersion=v1, code=409, details=StatusDetails(causes=[], group=coordination.k8s.io, kind=leases, name=strimzi-cluster-operator, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=Operation cannot be fulfilled on leases.coordination.k8s.io "strimzi-cluster-operator": the object has been modified; please apply your changes to the latest version and try again, metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Conflict, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:34:40 INFO  ClusterOperator:140 - Triggering periodic reconciliation for namespace ksql043
2025-09-22 16:34:40 WARN  VersionUsageUtils:60 - The client is using resource type 'kafkamirrormakers' with unstable version 'v1beta2'
2025-09-22 16:34:41 WARN  VersionUsageUtils:60 - The client is using resource type 'kafkabridges' with unstable version 'v1beta2'
2025-09-22 16:34:41 WARN  VersionUsageUtils:60 - The client is using resource type 'kafkarebalances' with unstable version 'v1beta2'
2025-09-22 16:34:41 WARN  VersionUsageUtils:60 - The client is using resource type 'kafkaconnectors' with unstable version 'v1beta2'
2025-09-22 16:34:41 INFO  AbstractOperator:264 - Reconciliation #5(timer) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:34:41 ERROR AbstractOperator:283 - Reconciliation #5(timer) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:34:41 WARN  AbstractOperator:556 - Reconciliation #5(timer) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:36:40 INFO  ClusterOperator:140 - Triggering periodic reconciliation for namespace ksql043
2025-09-22 16:36:41 INFO  AbstractOperator:264 - Reconciliation #6(timer) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:36:41 ERROR AbstractOperator:283 - Reconciliation #6(timer) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:36:41 WARN  AbstractOperator:556 - Reconciliation #6(timer) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:38:40 INFO  ClusterOperator:140 - Triggering periodic reconciliation for namespace ksql043
2025-09-22 16:38:41 INFO  AbstractOperator:264 - Reconciliation #7(timer) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:38:41 ERROR AbstractOperator:283 - Reconciliation #7(timer) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:38:41 WARN  AbstractOperator:556 - Reconciliation #7(timer) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:40:40 INFO  ClusterOperator:140 - Triggering periodic reconciliation for namespace ksql043
2025-09-22 16:40:41 INFO  AbstractOperator:264 - Reconciliation #8(timer) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:40:41 ERROR AbstractOperator:283 - Reconciliation #8(timer) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:40:41 WARN  AbstractOperator:556 - Reconciliation #8(timer) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:42:40 INFO  ClusterOperator:140 - Triggering periodic reconciliation for namespace ksql043
2025-09-22 16:42:40 INFO  AbstractOperator:264 - Reconciliation #9(timer) Kafka(ksql043/my-cluster): Kafka my-cluster will be checked for creation or modification
2025-09-22 16:42:41 ERROR AbstractOperator:283 - Reconciliation #9(timer) Kafka(ksql043/my-cluster): createOrUpdate failed
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
2025-09-22 16:42:41 WARN  AbstractOperator:556 - Reconciliation #9(timer) Kafka(ksql043/my-cluster): Failed to reconcile
io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:507) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleResponse(OperationSupport.java:524) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.handleGet(OperationSupport.java:467) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.handleGet(BaseOperation.java:792) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.requireFromServer(BaseOperation.java:193) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:149) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.get(BaseOperation.java:98) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$0(ContextImpl.java:178) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76) ~[io.vertx.vertx-core-4.5.9.jar:4.5.9]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[io.netty.netty-common-4.1.111.Final.jar:4.1.111.Final]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://172.30.0.1:443/apis/apps/v1/namespaces/ksql043/statefulsets/my-cluster-kafka. Message: statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043". Received status: Status(apiVersion=v1, code=403, details=StatusDetails(causes=[], group=apps, kind=statefulsets, name=my-cluster-kafka, retryAfterSeconds=null, uid=null, additionalProperties={}), kind=Status, message=statefulsets.apps "my-cluster-kafka" is forbidden: User "system:serviceaccount:ksql043:strimzi-cluster-operator" cannot get resource "statefulsets" in API group "apps" in the namespace "ksql043", metadata=ListMeta(_continue=null, remainingItemCount=null, resourceVersion=null, selfLink=null, additionalProperties={}), reason=Forbidden, status=Failure, additionalProperties={}).
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:660) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:640) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:589) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:549) ~[io.fabric8.kubernetes-client-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$completeOrCancel$10(StandardHttpClient.java:142) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147) ~[?:?]
	at io.fabric8.kubernetes.client.utils.AsyncUtils.lambda$retryWithExponentialBackoff$3(AsyncUtils.java:90) ~[io.fabric8.kubernetes-client-api-6.13.3.jar:?]
	at java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841) ~[?:?]
	at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
	at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
	at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	... 1 more
